src/util.ts

```diff
@@ -26,3 +26,22 @@ export const urlWithContextAsQuery = (url: URL, context: IContext) => {
         });
     return urlWithQuery;
 };
+
+const sortObjectProperties = (
+    obj: Record<string, any>
+): Record<string, any> => {
+    const sortedKeys = Object.keys(obj).sort();
+    const sortedObj: Record<string, any> = {};
+    sortedKeys.forEach((key) => {
+        if (typeof obj[key] === 'object') {
```

Comment:
`null` est de type `Object` 😅 
```suggestion
        if (obj[key] !== null && typeof obj[key] === 'object') {
```